### PR TITLE
adds 1 gbp to armhulen's account (grammar fix)

### PR DESCRIPTION
**Pull request by ghost**
in every other case it uses our instead of your, so it's weird that it doesn't do that here. I kept it for the latter part of the removal thing, because it makes sense. You aren't a ling anymore.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
You did it!

File: code/datums/mind.dm [hunk 1]
```diff
@@ -1031,12 +1031,12 @@
 			if("clear")
 				remove_antag_datum(/datum/antagonist/changeling)
 				special_role = null
-				to_chat(current, "<span class='userdanger'>You grow weak and lose your powers! You are no longer a changeling and are stuck in your current form!</span>")
+				to_chat(current, "<span class='userdanger'>We grow weak and lose our powers! You are no longer a changeling and are stuck in your current form!</span>")
```
Comment (ShizCalev) [inline code/datums/mind.dm]: Wouldn't it be "We are no longer a changeling"?

Comment (Armhulen) [reply code/datums/mind.dm]: he's not a changeling

**Comment by Armhulen** [comment]
image
J-just do grammar fixes

File: code/datums/mind.dm (hunk 1 above)
Comment (RemieRichards) [reply code/datums/mind.dm]: no, this entire line should be reverted, it's happened AFTER you're no longer a changeling, thus it SHOULD be You and Your

File: code/datums/mind.dm [hunk 2]
```diff
@@ -1031,12 +1031,12 @@
 			if("clear")
 				remove_antag_datum(/datum/antagonist/changeling)
 				special_role = null
-				to_chat(current, "<span class='userdanger'>You grow weak and lose your powers! You are no longer a changeling and are stuck in your current form!</span>")
+				to_chat(current, "<span class='userdanger'>We grow weak and lose our powers! You are no longer a changeling and are stuck in your current form!</span>")
 				message_admins("[key_name_admin(usr)] has de-changeling'ed [current].")
 				log_admin("[key_name(usr)] has de-changeling'ed [current].")
 			if("changeling")
 				var/datum/antagonist/changeling/C = make_Changling()
-				to_chat(current, "<span class='boldannounce'>Your powers are awoken. A flash of memory returns to us...we are [C.changelingID], a changeling!</span>")
```
Comment (DoctoreMad) [inline code/datums/mind.dm]: This removes important lore regarding Egypt, specially regarding the racial distribution of the people in royalty.

File: code/datums/mind.dm (hunk 1 above)
Comment (Armhulen) [reply code/datums/mind.dm]: Ok

File: code/datums/mind.dm (hunk 2 above)
Comment (Armhulen) [reply code/datums/mind.dm]: Thanks for reviewing my code!

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**